build.gradle

```diff
@@ -42,7 +42,7 @@ task coverage(type: JacocoReport) {
 
 dependencies {
     String jUnitVersion = '5.4.0'
-    String javaFxVersion = '17.0.7'
+    String javaFxVersion = '17.0.11'
```

Comment:
JavaFX 17.0.11 is the latest version of JavaFX 17, but we should consider upgrading it to JavaFX 21 eventually, as it has long-term support. Source: https://gluonhq.com/products/javafx/